h2o-algos/src/main/java/hex/schemas/TreeV3.java

```diff
@@ -0,0 +1,30 @@
+package hex.schemas;
+
+import water.Iced;
+import water.api.API;
+import water.api.schemas3.KeyV3;
+import water.api.schemas3.SchemaV3;
+
+public class TreeV3 extends SchemaV3<Iced, TreeV3> {
```

Comment:
We should also indicate how do we treat NAs (do they go left/right, is the split NA vs Rest and so on)

Reply:
Resolved. This format required does not indicate whether one of the edges to the node's children allows NAs. It basically says if NA can reach the node. This enables the client parsing it to determine the way NAs are handled:

- None of the children has NA allowed
- One of the children has Na allowed
- If one of the children has NA allowed, which one it is (left, right - based on the array it is situated in).